cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -470,11 +470,11 @@ private void dropDataset(DatasetId instance, DatasetSpecification spec) throws E
     privilegesManager.revoke(instance);
   }
 
-  private void disableExplore(DatasetId datasetInstance) {
+  private void disableExplore(DatasetId datasetInstance, DatasetSpecification spec) {
     // Disable ad-hoc exploration of dataset
     // Note: today explore enable is not transactional with dataset create - CDAP-8
     try {
-      exploreFacade.disableExploreDataset(datasetInstance);
+      exploreFacade.disableExploreDataset(datasetInstance, spec);
     } catch (Exception e) {
       LOG.error("Cannot disable explore for dataset instance {}", datasetInstance, e);
```

Comment:
Should this message be (being consistent to indicate that "Explore" is significant:
```
LOG.error("Cannot disable Explore for dataset instance {}",